__tests__/intergration.test.js

```diff
@@ -7,6 +7,24 @@ const app = require("../app");
 beforeEach(() => seed(data));
 afterAll(() => db.end());
 
+//__________________ Section: /api _______________//
+describe("GET: /api", () => {
+  test("200: responds with a list of api endpoints available", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body: { api_endpoints } }) => {
+        const parsedData = JSON.parse(api_endpoints);
+        const cleanEndpoints = Object.values(parsedData).slice(1);
+        cleanEndpoints.forEach((endpoint) => {
+          expect(endpoint).toHaveProperty("description");
+          expect(endpoint).toHaveProperty("queries");
+          expect(endpoint).toHaveProperty("exampleResponse");
+        });
```

Comment:
This is a good start, but is a little over complicated than it needs to be. The idea of this test is to ensure what we get back is the contents from the endpoints.json file. We dont actually need to worry about specifically what the file contains at this stage. The easiest way to test for this is requiring the endpoints.json file in this test file and asserting that api_endpoints is the same as the required endpoints.json file contents.